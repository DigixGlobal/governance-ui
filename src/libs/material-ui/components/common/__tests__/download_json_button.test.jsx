import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import DownloadJsonButton from '../download_json_button';

describe('<DownloadJsonButton />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <DownloadJsonButton getFileInfo={jest.fn} />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
