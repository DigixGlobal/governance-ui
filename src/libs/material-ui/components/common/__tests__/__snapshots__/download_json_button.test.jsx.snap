// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DownloadJsonButton /> renders correctly 1`] = `
<Button
  as="button"
  onClick={[Function]}
/>
`;
